goodwhiskers/app/controllers/products_controller.rb

```diff
@@ -29,6 +29,13 @@ def edit
   # POST /products.json
   def create
     @product = Product.new(product_params)
```

Comment:
following line sends an object from the hash to cloudinary.  The response from cloudinary is saved in 'response' variable.